assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -0,0 +1,173 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import RadioToggle from 'components/radioToggle/radioToggle';
+import NumberInput from 'components/numberInput/numberInput';
+import { clickSubstituteKeyPressHandler } from 'helpers/utilities';
+import type { Contrib, ContribError, Amounts } from 'helpers/contributions';
+
+
+// ----- Setup ----- //
+
+const amountToggles = {
+  recurring: {
+    name: 'contributions-amount-recurring-toggle',
+    radios: [
+      {
+        value: '5',
+        text: '£5',
+      },
+      {
+        value: '10',
+        text: '£10',
+      },
+      {
+        value: '20',
+        text: '£20',
+      },
+    ],
+  },
+  oneOff: {
+    name: 'contributions-amount-oneoff-toggle',
+    radios: [
+      {
+        value: '25',
+        text: '£25',
+      },
+      {
+        value: '50',
+        text: '£50',
+      },
+      {
+        value: '100',
+        text: '£100',
+      },
+      {
+        value: '250',
+        text: '£250',
+      },
+    ],
+  },
+};
+
+const contribToggle = {
+  name: 'contributions-period-toggle',
+  radios: [
+    {
+      value: 'RECURRING',
+      text: 'Monthly',
+    },
+    {
+      value: 'ONE_OFF',
+      text: 'One-off',
+    },
+  ],
+};
+
+const contribErrors = {
+  tooLittleRecurring: 'Please enter at least £5',
+  tooLittleOneOff: 'Please enter at least £1',
+  tooMuch: 'We are presently only able to accept contributions of £2000 or less',
+  noEntry: 'Please enter a numeric amount',
+};
+
+
+// ----- Types ----- //
+
+// Disabling the linter here because it's just buggy...
+/* eslint-disable react/no-unused-prop-types */
+
+type PropTypes = {
+  contribAmount: Amounts,
+  contribType: Contrib,
+  contribError: ?ContribError,
+  changeContribRecurringAmount: (string) => void,
+  changeContribOneOffAmount: (string) => void,
+  changeContribAmount: (string) => void,
+  toggleContribType: (string) => void,
+  onNumberInputKeyPress: () => void,
+};
+
+/* eslint-enable react/no-unused-prop-types */
+
+
+// ----- Functions ----- //
+
+function errorMessage(error: ?ContribError) {
+
+  if (error) {
```

Comment:
Does this error message has the same format of ErrorMessage component? If so, we can reuse that component here.